.gitignore

```diff
@@ -1,7 +1,11 @@
-dist/
```

Comment:
We don't want to remove the `dist` directory from this list, or the `.DS_Store` or `.idea` below.